src/repos/ArticleRepo.ts

```diff
@@ -1,28 +1,51 @@
 import Filter from 'bad-words';
 import { ObjectId } from 'mongodb';
 import { Article, ArticleModel } from '../entities/Article';
-import { DEFAULT_LIMIT, MAX_NUM_DAYS_OF_TRENDING_ARTICLES } from '../common/constants';
+import { DEFAULT_LIMIT, MAX_NUM_DAYS_OF_TRENDING_ARTICLES, IS_FILTER_ACTIVE, FILTERED_WORDS} from '../common/constants';
 import { PublicationModel } from '../entities/Publication';
 
+function isArticleFiltered(article: Article){
+  if(IS_FILTER_ACTIVE){
```

Comment:
missing white spaces

is there no auto linter?

Reply:
I don't see a pre-commit hook, that's for another PR another time I can look into one too since it's not high priority.